src/http/get-ipr-000user-000repo-pull-000prEtc/check-form.js

```diff
@@ -83,7 +83,7 @@ const authorsP = request(branchURL).then((json) => JSON.parse(json)).then((data)
 
 const teamURL = 'https://api.github.com/orgs/tc39/teams/delegates';
 
-function getMembers(teamID, page = 1) {
+const getMembers = (teamID, page = 1) => {
```

Comment:
why this change?

Reply:
All changes related to converting function expressions this way were to satisfy linter warnings I got from the current linting config (`"@ljharb/eslint-config": "^21.1.0`, `"eslint": "=8.8.0"`; I'm guessing perhaps it's this: https://github.com/ryanblock/tc39-ci/blob/f5c7d191712c114d2f987eb9047f6d588d0118f3/.eslintrc.js#L17

Would you like to see all those converted back? I don't personally care, I just assumed that's how you wanted it!